runtime/Go/antlr/atn.go

```diff
@@ -60,16 +60,16 @@ func NewATN(grammarType int, maxTokenType int) *ATN {
 //  If {@code ctx} is nil, the set of tokens will not include what can follow
 //  the rule surrounding {@code s}. In other words, the set will be
 //  restricted to tokens reachable staying within {@code s}'s rule.
-func (this *ATN) NextTokensInContext(s ATNState, ctx RuleContext) *IntervalSet {
-	var anal = NewLL1Analyzer(this)
-	var res = anal.LOOK(s, nil, ctx)
+func (a *ATN) NextTokensInContext(s ATNState, ctx RuleContext) *IntervalSet {
+	var anal = NewLL1Analyzer(a)
+	var res = anal.look(s, nil, ctx)
```

Comment:
Good catch. I looked for uses of LOOK in the runtime, and they were all internal, but it looks like LOOK is public in Java. I'll rename it to Look.